<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookReceipts</class>
 <widget class="QDialog" name="BookReceipts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Library Receipts</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date / Time: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="dateTimeLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sold By : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="seller">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>309</x>
     <y>10</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Sale Information</string>
   </property>
   <widget class="QComboBox" name="transactionType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="department">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="transactionCode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="firstName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Qty : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="quantity">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="total">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="description">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Unit Price : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="unitPrice">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="printReceiptCheckBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Print Receipt</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="fineCheckBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>This receipt is for a fine</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>seller</tabstop>
  <tabstop>firstName</tabstop>
  <tabstop>lastName</tabstop>
  <tabstop>quantity</tabstop>
  <tabstop>unitPrice</tabstop>
  <tabstop>total</tabstop>
  <tabstop>transactionType</tabstop>
  <tabstop>department</tabstop>
  <tabstop>transactionCode</tabstop>
  <tabstop>description</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>printReceiptCheckBox</tabstop>
  <tabstop>submitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
